packages/notification-services-controller/CHANGELOG.md

```diff
@@ -44,12 +50,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.12.1]
 
-### Uncategorized
```

Comment:
This changelog update was left uncategorised, updated this.

NOTE - is it possible for us to add this as part of the changelog validation/linting?

Reply:
Hmm, would need to dig into [auto-changelog](https://github.com/cookpete/auto-changelog) to determine if this is an available setting or if it would need to be a custom rule.